gen/plugins/com.salesforce/com.salesforce.plugin.smartstore.js

```diff
@@ -30,6 +30,8 @@ var SERVICE = "com.salesforce.smartstore";
 
 var exec = require("com.salesforce.util.exec").exec;
 
+var defaultStoreName = "defaultStore";
```

Comment:
This has to match what's defined in `native`, correct? Can we remove this, so `native` is the single source of truth and we don't have to maintain them in both?